### PR TITLE
[ZH] Update `Client/Beatmap editor`

### PR DESCRIPTION
**Pull request by Sunnyboy971**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Client/Beatmap_editor/zh.md [hunk 1]
```diff
@@ -1,22 +1,42 @@
...
-- [Design](Design)：这就是故事板设计，多数作图者只会在这里使用“背景图片/视频”按钮。
-- [Timing](Timing)：这是设置乐曲的 BPM 和 Offset。这个窗口还控制点击音效。
-- [Song setup](Song_Setup)：这是关于地图的各种选项进行设置。这包括颜色和难度设置。
+谱面编辑器被分为4个主要部分，可以在编辑器左上角看到：
 
-## Related articles
+- Compose
+- Design
+- Timing
+- Song setup
```
Comment (YumeMuzi) [inline wiki/Client/Beatmap_editor/zh.md:14]: 可以改成中文 (英文) 的方式，保留原文的同时将翻译的优先级提前，下文同理

Comment (YumeMuzi) [reply wiki/Client/Beatmap_editor/zh.md:14]: 包括副标题也是一样

**Review by YumeMuzi — CHANGES_REQUESTED** [review]
改掉那些comment就差不多了

**Review by YumeMuzi — APPROVED** [review]
seems great

**Review by TicClick — APPROVED** [review]

**Pull request merged**